lib/comfortable_mexican_sofa/extensions/is_taggable.rb

```diff
@@ -82,12 +74,12 @@ def unkeyword!(keyword_ids:)
       # Synchorize the taggings of this item with the ones in its mirrors. So all of them
       # are associated to the same tags
       def sync_mirrors_with_taggings
-        if is_mirrored_with_taggings
-          mirrors.each { |mirror| mirror.assign_new_keywords!(new_keyword_ids: keywords.map(&:id)) }
+        return unless mirrored_with_taggings?
+
+        mirrors.each do |mirror|
+          mirror.assign_new_keywords!(new_keyword_ids: keywords.map(&:id))
```

Comment:
If `keywords` is a HM or HABTM association, just using `keyword_ids` would be nicer and more efficient than mapping over the `id` as this does now.